rclrs/src/rcl_utils.rs

```diff
@@ -0,0 +1,101 @@
+use crate::rcl_bindings::*;
+use crate::{RclrsError, ToResult};
+
+use libc::c_void;
+use std::ptr::null_mut;
+
+/// Trait used for obtaining information from rcl_arguments_t.
+pub(crate) trait RclGetArg {
+    /// Get count function from rcl library (e.g. rcl_arguments_get_count_unparsed)
+    unsafe fn get_count(rcl_args: *const rcl_arguments_t) -> ::std::os::raw::c_int;
+
+    /// Get function from rcl library (e.g. rcl_arguments_get_unparsed)
+    unsafe fn get_indices(
+        rcl_args: *const rcl_arguments_t,
+        allocator: rcl_allocator_t,
+        out_ptr: *mut *mut ::std::os::raw::c_int,
+    ) -> rcl_ret_t;
```

Comment:
Is it possible/desirable to have this return a `Result` instead? Akin to how we use `.ok()?` for most of our calls to `rcl`?

Reply:
It was intended to be just a wrapper around corresponding `rcl_argument_get...` and `rcl_argument_get_count...`, so i left the interface repeated.